.env.example

```
@@ -1,20 +1,74 @@
+PUBLIC_VERIFF_KEY="your-veriff-key"
+VERIFF_HMAC_KEY="your-veriff-hmac-key"
+
+# set this to allow or deny
+DUPLICATES_POLICY="DENY"
+IP_ADDR="localhost"
+
+PICTIQUE_DATABASE_URL=postgresql://postgres:postgres@localhost:5432/pictique
+
+PICTIQUE_MAPPING_DB_PATH=/path/to/pictique/mapping/db
+BLABSY_MAPPING_DB_PATH="/path/to/blabsy/mapping/db"
+DREAMSYNC_MAPPING_DB_PATH="/path/to/dreamsync/mapping/db"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)